### DISCUSSION TITLE
Date/DateTime Picker need to improve

### DISCUSSION THREAD

**Discussion by thphuoc** [bosskmk/pluto_grid#441] [Ideas]
Expect improve: 
1. Display focusing date on scroll: The title should display for current date is focusing. Default focusing date should be current day of current month or first day of other months.
2. Date/Datetime picker should allow quick select month/year.
3. UI of Date/DateTime picker should change to Material Design
4. Allow input date time along with select on calendar

**Comment by bosskmk** [comment]
@thphuoc 
Some of you mentioned some of the features you mentioned were updated and distributed to 3.1.0.

I will try to add the function to enter the date directly.

Thank you for your opinion.

https://weblaze.dev/pluto_grid/build/web/#feature/date-type-column
